symbolic-symcache/src/format.rs

```diff
@@ -298,7 +299,7 @@ pub struct HeaderV2 {
     pub preamble: Preamble,
     /// Debug identifier of the object file.
     pub debug_id: DebugId,
-    /// CPU architecture of the debug file.
+    /// CPU architecture of the debug file. NOTE: debug file or object file?
```

Comment:
I think in theory Debug Information File is a larger group which include Object Files.  It's pretty interchangeable here i reckon, but object file would be more consistent I guess.